### ISSUE TITLE
Top 10 Things To Add

### ISSUE THREAD

**Issue by Bipbip2109** [xCollateral/VulkanMod#408]
![Top Cinco](https://github.com/xCollateral/VulkanMod/assets/88253359/cd8dbc9b-1c5d-4c48-8841-2587f7c3dc2b)

Hello guys, today we have a top Cinco epic things to add to the hit mod Vulkanmod :

Uno 1    ] Anti-Aliasing methods (FXAA, TAA, SSAA, MSAA, MLAA, CMAA...)
Dos 2     ] Texture Filtering methods (Bilinear, Trilinear, Anisotropic...)
Tres 3     ] More Mipmap levels
Cuatro 4] Mesh Shaders (VK_EXT_mesh_shader)
Cinco 5  ] Upscaling methods (FSR, DLSS, XESS, CAS, TAAU...)

**Comment by Bipbip2109** [comment]
There are not actually 10 things I lied

**Comment by AlexanderwithaA** [comment]
I don’t think vkmod needs fsr compat built in and afaik mc isn’t fill limited. That’d probably help on ray traced games

**Comment by NXTler** [comment]
Why do you specifically want 'Mesh Shaders'?

**Comment by lv37** [comment]
> Why do you specifically want 'Mesh Shaders'?

https://github.com/xCollateral/VulkanMod/issues/257

**Comment by NXTler** [comment]
I don't think it is as important as it seems. There is still **a lot** of performance left with much simpler methods, like greedy meshing or LODs, that are not as hard to implement.

**Comment by Bipbip2109** [comment]
> I don't think it is as important as it seems. There is still **a lot** of performance left with much simpler methods, like greedy meshing or LODs, that are not as hard to implement.

LOD looks horrible in Minecraft since the game already has pretty much the lowest amount of vertices possible per block and merging multiple blocks together will just look awful if you pay any sort of attention to the screen in front of you (Example: Distant Horizons).

Greedy meshing would be nice tho

**Comment by blankuserrr** [comment]
> > I don't think it is as important as it seems. There is still **a lot** of performance left with much simpler methods, like greedy meshing or LODs, that are not as hard to implement.
> 
> LOD looks horrible in Minecraft since the game already has pretty much the lowest amount of vertices possible per block and merging multiple blocks together will just look awful if you pay any sort of attention to the screen in front of you (Example: Distant Horizons).
> 
> Greedy meshing would be nice tho

true, LODs look fine until you install a zoom mod.

**Comment by Felix14-v2** [comment]
> LODs look fine until you install a zoom mod.

LODs look fine even with a zoom mod if we speak about LODs in Voxy. Distant Horizons is a bad example.

**Comment by temotskipa** [comment]
#495

Ray tracing support would also be great.

**Comment by blankuserrr** [comment]
> I don't think it is as important as it seems. There is still **a lot** of performance left with much simpler methods, like greedy meshing or LODs, that are not as hard to implement.

I thought minecraft devs explicitly chose to not greedy mesh

**Comment by NXTler** [comment]
I can only imagine them avoiding greedy meshing because it's a lot of work to get right. If you do greedy meshing properly, you gain a lot of performance.
I'm also curious where you got that information?
